### PR TITLE
Add support for CHECK Constraints

### PR DESCRIPTION
**Pull request by aroralakshya**
Add support for CHECK Constraints
#167 
Signed-Off-By: Lakshay arora <arora.lakshya123@gmail.com>

### CODE REVIEW

**Comment by williamdes** [comment]
Code style of `if(` must be `if (`
Is `./test/ci-lint` also broken for you ?

**Comment by devenbansod** [comment]
Hi @nulll-pointer thanks for the PR. Could you please fix the breaking test?

And also, please add tests to cover this new syntax.

**Comment by williamdes** [comment]
@nulll-pointer Can you please update the PR ?

File: src/Components/Check.php [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+     * @var logicalOperatorsArray
+     */
+    public $logicalOperators;
+
+    /**
+     * Possible operators in the component.
+     *
+     * @var OperatorsArray
+     */
+    public static $operatorsArray = array("=", ">", "<", "<=", ">=", "!=", "LIKE", "NOT", "BETWEEN", "IS", "NULL", "REGEXP", "REGEXP");
```
Comment (bigfoot90) [inline src/Components/Check.php]: `REGEXP` duplicated value in array

**Comment by williamdes** [comment]
I do not like to have to close this PR, but I have to.
Needs more unit tests and to be done onto QA

**Comment by williamdes** [comment]
But still, thank you @aroralakshya !

**Pull request closed without merging**